tests/resources/functions/cpp/index.cc

```diff
@@ -1,66 +0,0 @@
-#include <iostream>
-#include <string>
-#include <curl/curl.h>
-
-static size_t WriteCallback(void *contents, size_t size, size_t nmemb, void *userp)
```

Comment:
any reason we deleted this file ?

Reply:
C++ wasnt finished when I started this branch.

Re-added CPP tests.